.github/workflows/reusable-python-build_poetry_package.yml

```diff
@@ -0,0 +1,122 @@
+name: Build poetry package with Artifactory 
```

Comment:
Artifactory can be optional: would build without it too if it does not need to fetch packages from it